components/cam/bld/configure

```diff
@@ -958,11 +958,11 @@ my $clubb_sgs = $cfg_ref->get('clubb_sgs');
 
 # consistency checks...
 
-# CLUBB_SGS only works with mg microphysics
```

Comment:
if you could add a similar section checking to make sure SHOC (SHOC_SGS) is configured with MG2 or P3 that would be great.  I wanted to add this in my SHOC commit but couldn't since P3 wasn't a configure option.

Reply:
done, can you check if this is what you want in the new version?

Reply:
looks good.  just minor detail that the comment is inconsistent.

Reply:
oh yeah, I'll fix that